### PR TITLE
Load meta from external file

### PR DESCRIPTION
**Pull request by namse**
# What I needed and what I did
I wanted to save metadata like "Character's color", etc. I think it's not a good idea to save data in code, so I made one.
I considered to reuse this system in the luda game, not only editor.

Design docs
https://docs.google.com/document/d/1hv3mmxQhKVRjxnHvo1spnyKkgK8OOxashbLY0T867Eo/edit#

### CODE REVIEW

File: luda-editor/luda-editor-client/src/app/sequence_list/mod.rs [hunk 1]
```diff
@@ -170,8 +161,6 @@ fn calculate_sequence_duration(
                 .clips
                 .iter()
                 .fold(duration, |duration, clip| duration.max(clip.end_at)),
-            Track::Subtitle(track) => track.clips.iter().fold(duration, |duration, clip| {
-                duration.max(clip.end_at(namui::Language::Ko, subtitle_play_duration_measurer))
-            }),
+            Track::Subtitle(track) => Time::zero(),
```
Comment (namse) [inline luda-editor/luda-editor-client/src/app/sequence_list/mod.rs:164]: This is the break changes. I think it's not necessary to calculate duration of subtitle. If you think it's not a good idea, I can revert it.

File: luda-editor/luda-editor-client/src/app/editor/clip_select.rs [hunk 2]
```diff
@@ -383,6 +385,17 @@ mod tests {
             context_menu: None,
             sequence_saver: SequenceSaver::new("", sequence.clone(), socket.clone()),
             sheet_sequence_syncer: SheetSequenceSyncer::new(""),
+            meta_container: Arc::new(MetaContainer::new(
```
Comment (namse) [inline luda-editor/luda-editor-client/src/app/editor/clip_select.rs:388]: This is mocking for test.

File: luda-editor/luda-editor-client/src/app/app.rs [hunk 3]
```diff
@@ -1,43 +1,60 @@
...
     type Props = ();
     fn render(&self, _: &Self::Props) -> namui::RenderingTree {
-        let screen_size = namui::screen::size();
-        self.router.render(&RouterProps {
-            screen_wh: Wh {
-                width: screen_size.width as f32,
-                height: screen_size.height as f32,
-            },
-        })
+        match self.meta_container.get_meta() {
```
Comment (namse) [inline luda-editor/luda-editor-client/src/app/app.rs:23]: If meta_container didn't load meta yet, it won't render. I didn't process about the error that occurs during meta loading.

File: luda-editor/luda-editor-client/src/app/editor/mod.rs [hunk 4]
```diff
@@ -406,7 +409,7 @@ impl namui::Entity for Editor {
                 job: &self.job,
                 selected_clip_ids: self.selected_clip_ids.iter().collect::<Vec<_>>().as_slice(),
                 sequence: self.get_sequence(),
-                subtitle_play_duration_measurer: &self.subtitle_play_duration_measurer,
+                subtitle_play_duration_measurer: &self.get_meta(),
```
Comment (bigfoodK) [inline luda-editor/luda-editor-client/src/app/editor/mod.rs:412]: Are you going to keep using this name? `subtitle_play_duration_measurer`

Comment (bigfoodK) [reply luda-editor/luda-editor-client/src/app/editor/mod.rs:412]: Oh, it makes sense. Meta can be measurer!

**Review by bigfoodK — APPROVED** [review]

**Pull request merged**